### PR TITLE
Add ev.Item to ProcessingHotkeyEventArgs

### PR DESCRIPTION
**Pull request by steven4547466**

### CODE REVIEW

File: Exiled.Events/EventArgs/ProcessingHotkeyEventArgs.cs [hunk 1]
```diff
@@ -40,6 +43,11 @@ public ProcessingHotkeyEventArgs(Player player, HotkeyButton hotkey, bool isAllo
         /// </summary>
         public HotkeyButton Hotkey { get; }
 
+        /// <summary>
+        /// Gets or sets the item change to.
+        /// </summary>
+        public Item Item { get; set;  }
```
Comment (joker-119) [inline Exiled.Events/EventArgs/ProcessingHotkeyEventArgs.cs:49]: The setter for this should check if the player actually possesses the item, and either throw an invalid operation error, or give them the item if they do not have it.

**Review by joker-119 — CHANGES_REQUESTED** [review]

**Comment by TheUltiOne** [comment]
Did Nao ever make this? Can this be closed?

**Comment by NaoUnderscore** [comment]
I didn't make it yet, because I was dealing with attachments.

**Comment by TheUltiOne** [comment]
yeah but shouldnt this be closed as you're "cloning" this? since it will be a different PR.

**Pull request closed without merging**